### PR TITLE
 CSI-4050 pyds8k 1.4.0

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: controller/array_action/ds8k_rest_client.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def __init__(self, service_address, user, password,
         client_kwargs = {'service_address': service_address,
                          'user': user,
                          'password': password,
+                         'verify': False,
```
Comment (oriyarde) [inline controller/array_action/ds8k_rest_client.py:66]: do we have a ticket to support verification?

Comment (ArbelNathan) [reply controller/array_action/ds8k_rest_client.py:66]: do we want to?

Comment (oriyarde) [reply controller/array_action/ds8k_rest_client.py:66]: sure

Comment (ArbelNathan) [reply controller/array_action/ds8k_rest_client.py:66]: can you open a ticket?

**Review by oriyarde — APPROVED** [review]
LGTM

File: controller/array_action/ds8k_rest_client.py (hunk 1 above)
Comment (oriyarde) [reply controller/array_action/ds8k_rest_client.py:66]: CSI-4103

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**